### ISSUE TITLE
Hashtags

### ISSUE THREAD

**Issue by guaka** [guaka/hitchticker#24]
- [x] create a search page that shows stuff, `?q=stuff`
- [x] turn #hashtags into links to `?q=#hashtags`

---
#21 @UserStefan: "Implementing groups to tweet for different issues (it is unsorted, need some more structure for better overview. also i would like to stream my own trip)."

**Comment by guaka** [comment]
How is this supposed to work?

**Comment by UserStefan** [comment]
You post a status and it is displayed in a specific group/ressort. Lets say we have DTSG Race, so people in the race post in DTSG Race group/ressort. Or we have OnMyWayToHitchgathering group, so people on their way to hitchgathering post there status in this group. 

If we have this, we also can display only this group-ticker on a certain place/homepage. Makes it nice to play with it. Like on Hitchwiki "People are actually going to the Hitchgathering, follow us on our journey." or "This is my personal Hitchticker which gives you short updates, where i am and what i am doing."

**Comment by UserStefan** [comment]
My idea is: When we have this feature, people might be more tempted to use the ticker in a creative way, play with it and have fun to ticker messages. This is a very basic idea, to make the thing living and not just being a dead application, that nobody cares about. I think tickers are a very funny thing and can make hitchhiking trips become exciting for the public.

If we have this, we can also call for ticker-gatherings and ask people to submit a tweet for a certain topic (like ticker here for some outstanding act of kidness, that you received by your driver). [Sorry, i am just writing some ideas down, which i would include in the hitchwiki article for giving some inspiration about how to use this thing and make people engage]. We also can have a "FAQ-Ticker" for short question and answering. Or an "Hitchwiki progress ticker". Whatever comes up. Whatever makes sense. Whatever people want.

It is like Twitter, but only for hitchhikers and nomads, much more basic and Open Source (which is most important. Non-Profit, ideologically clean and IF we can engage enough people, it exsts the potential to make something special out of it. Something funny, where we like to have a look into.)

**Comment by platschi** [comment]
Maybe something like using hashtags during specific events and have the ability to sort on the website by hashtag ( e.g. hitchticker.org/dtsg/,  hitchticker.org/hitchgathering/,) and so on? 

Would require participants to use that tag for each message, though.

**Comment by simison** [comment]
+1

A simple thing which can be used in sooo many ways.

**Comment by guaka** [comment]
Yes, hashtags were my first thought as well, just didn't want to push in that direction :)

**Comment by UserStefan** [comment]
Then go. Push more pls. We need to bring it to a point now, where we can kinda close the first develompent stage.

**Comment by UserStefan** [comment]
Can we close this?

I see it is working like hitchticker.info/hashtag/hitchgathering/ for #hitchgatherng
